src/components/NetworkSelector.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+interface NetworkSelectorProps {
+    network: string;
+    setNetwork: (network: string) => void;
+}
+
+export class NetworkSelector extends React.Component<NetworkSelectorProps, Object> {
+
+    constructor(props: NetworkSelectorProps) {
+        super(props);
+        this.state = {};
+        this.handleNetworkUpdate = this.handleNetworkUpdate.bind(this);
+    }
+
+    handleNetworkUpdate(event: React.ChangeEvent<HTMLSelectElement>) {
+        this.props.setNetwork(event.target.value);
+    }
+
+    public render() {
+        return (
+            <div id="network-selector">
+                <p>
+                Network:
+                <select value={this.props.network} onChange={this.handleNetworkUpdate}>
+                <option>zeronet</option>
```

Comment:
Ideally, I'd want to read all available networks and their corresponding settings from a config file, _a la_ https://github.com/Cryptonomic/Conseil/blob/master/src/main/resources/application.conf. Is there something like that in the JS/TS world?

Reply:
Hmm, I don't know the best practice way for it but one way would be to have an endpoint that serves that information and the app makes that request on startup. It might also be possible to use that in the webpack configuration when the app gets bundled...

Reply:
The endpoint is a great idea. I'll add the backend server URLs to the Webpack configuration.

Reply:
Opened https://github.com/Cryptonomic/Conseil/issues/77 for the back end work. Ticket #23 will do it for the front end.